test/index.html

```diff
@@ -21,6 +21,7 @@
       // Load and run all tests (.html, .js):
       WCT.loadSuites([
         'basic-test.html',
+        'light-dom.html'
```

Comment:
While we are at it, the tests should also run in shadow dom. E.g. `basic-test.html?shadow=1`